### ISSUE TITLE
Solving Paper Clip Icon React 

### ISSUE THREAD

**Issue by TauqeerAhmad5201** [TauqeerAhmad5201/react-host#1]
The issue is related to the icon we are attaching in the [DescriptionList.js](https://github.com/TauqeerAhmad5201/react-host/blob/main/src/components/DescriptionList.js#L43) 

Go to /src/components/DescriptionList.js and look for Paper Clip Icon.

**Comment by SuyashOP-2** [comment]
Icon is not rendering is thats the issue

**Comment by TauqeerAhmad5201** [comment]
> Icon is not rendering is thats the issue

Yes, the icon is not being rendered. Try to add-on using importing.
